Improve style (#3223)
fishtown-analytics/docs.getdbt.com: website/snippets/connect-starburst-trino/roles-starburst-enterprise.md

[comment: For context, the section title used for this snippet is "Roles in Starburst Enterprise" ]: # 

> When connecting to a Starburst Enterprise cluster with built-in access controls enabled, you are not able to provide the role as a suffix to the username, so the default role for the provided username is used instead.

Reviewer: ```suggestion
If connecting to a Starburst Enterprise cluster with built-in access controls enabled, you can't add the role as a suffix to the username, so the default role for the provided username is used instead.
```

Author: Updated
